draft-ietf-ccwg-rfc5033bis

```diff
@@ -103,23 +105,26 @@ congestion control algorithms and for the IETF community when evaluating whether
 a proposal is appropriate for publication in the RFC series and for
 deployment in the Internet.
 
-This document obsoletes the similarly titled {{?RFC5033}} that was
+This document obsoletes the similarly titled {{?RFC53}} that was
 published in 2007 as a Best Current Practice to evaluate alternate
 congestion control algorithms as Experimental or Proposed Standard RFCs.
 
 The IETF's standard congestion control algorithms have been shown to have
 performance challenges in various environments
 (e.g., high-speed networks, cellular and WiFi wireless technologies,
 long distance satellite links)
-and also for specific traffic workloads (VoIP, gaming, and videoconferencing).
+and also flows carrrying specific workloads (VoIP, gaming, and videoconferencing).
 
-In 2007, TCP was the dominant consumer of this work, and proposals were
-typically discussed in research groups, for example the
-Internet Congestion Control Research Group (ICCRG).
+In 2007, TCP was the dominant consumer of this work, and proposals
+were typically discussed in the Internet Congestion Control Research Group (ICCRG).
+The Datagream Congestrion Copntrol Protocol (DCCP)
+was developed as a method for developing congestion control algorithms for
+datagram flows.
```

Comment:
This DCCP sentence is weird and I'm not sure why it exists.

Reply:
Para re-written to capture that TCP, SCTP and DCCP all existed. They can inherit CC algorithms from the same base - SCTP normally does; DCCP could or could do different. Please check new para.